.github/workflows/rust.yml

```diff
@@ -0,0 +1,22 @@
+on: [push]
+
+name: CI
+
```

Comment:
Could this file be merged into the existing CI workflow?

Reply:
this can be removed as there is a better CI workflow in upstream